prow/config/config.yaml

```diff
@@ -407,20 +407,6 @@ presubmits:
           value: "TRUE"
         image: quay.io/metal3-io/capm3-unit:master
         imagePullPolicy: Always
-  - name: govet
-    always_run: true
-    decorate: true
-    spec:
-      containers:
-      - args:
-        - ./hack/govet.sh
-        command:
-        - sh
-        env:
-        - name: IS_CONTAINER
-          value: "TRUE"
-        image: quay.io/metal3-io/capm3-unit:master
-        imagePullPolicy: Always
```

Comment:
This is CAPM3 repo, you need to remove govet from https://github.com/metal3-io/project-infra/blob/ebe2d90c26bb3bf284fd34c2c171a82058249da6/prow/config/config.yaml#L252-L265.